pkg/operator/starter.go

```diff
@@ -190,6 +190,9 @@ var RevisionConfigMaps = []revision.RevisionResource{
 	{Name: "kube-apiserver-server-ca", Optional: true},
 	{Name: "kubelet-serving-ca"},
 	{Name: "sa-token-signing-certs"},
+
+	// this is a copy of trusted-ca-bundle CM but with key modified to "tls-ca-bundle.pem" so that we can mount it the way we need
+	{Name: "trusted-ca-bundle"},
```

Comment:
I think you need this in `CertConfigMaps` /cc @deads2k

Reply:
I guess you're right, in the logs, I can see that the dirs created are:
```
"/etc/kubernetes/static-pod-resources/kube-apiserver-certs/configmaps/client-ca"
```
but on the other hand also
```
"/etc/kubernetes/static-pod-resources/kube-apiserver-pod-2/configmaps/trusted-ca-bundle"
```
which would explain why the `Directory` hostMount assumption fails, I'll move this to the `CertConfigMaps` then.